### PR TITLE
Update basic-availability-groups-always-on-availability-groups.md

### PR DESCRIPTION
**Pull request by pituach**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@pituach : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/database-engine/availability-groups/windows/basic-availability-groups-always-on-availability-groups.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ ms.author: chadam
...
 [!INCLUDE [SQL Server](../../../includes/applies-to-version/sqlserver.md)]
 
-  Always On Basic Availability Groups provide a high availability solution for SQL Server 2016 and SQL Server 2017 Standard Edition. A basic availability group supports a failover environment for a single database. It is created and managed much like traditional (advanced) [Always On Availability Groups &#40;SQL Server&#41;](../../../database-engine/availability-groups/windows/always-on-availability-groups-sql-server.md) with Enterprise Edition. The differences and limitations of basic availability groups are summarized in this document.  
+  Always On Basic Availability Groups provide a high availability solution for SQL Server from version 2016 and and above on Standard Edition. A basic availability group supports a failover environment for a single database. It is created and managed much like traditional (advanced) [Always On Availability Groups &#40;SQL Server&#41;](../../../database-engine/availability-groups/windows/always-on-availability-groups-sql-server.md) with Enterprise Edition. The differences and limitations of basic availability groups are summarized in this document.
```
Comment (cawrites) [inline docs/database-engine/availability-groups/windows/basic-availability-groups-always-on-availability-groups.md:17]: @pituach please update the syntax in this sentence:  Always On Basic Availability Groups provide a high availability solution for SQL Server from version 2016 **and and** above on Standard Edition.

**Review by cawrites — CHANGES_REQUESTED** [review]
please review comment

**Comment by cawrites** [comment]
@pituach we are advising you that we have updated this request with our own update as we have not heard back from you. The change will be updated in the next content refresh. Thank you for contacting Microsoft.

**Comment by cawrites** [comment]
#please-close

**Comment by pituach** [comment]
Sry I did not got the message about your request to update the syntax in my content (the double "and" which I wrote by mistake). I only got the message about closing the ticket now. Anyhow it is nice to see my name in the doc but this is not the important point. The important is that the document was fixed for the sake of the readers 🙂

File: docs/database-engine/availability-groups/windows/basic-availability-groups-always-on-availability-groups.md (hunk 1 above)
Comment (pituach) [reply docs/database-engine/availability-groups/windows/basic-availability-groups-always-on-availability-groups.md:17]: I saw this comment only now. The email probably got to the junk folder

**Comment by cawrites** [comment]
@pituach your contribution to update the content for our readers is appreciated. Thank you

**Pull request closed without merging**